.travis.yml

```diff
@@ -29,6 +29,8 @@ before_install:
   # Install dependencies of facter
   - wget https://s3.amazonaws.com/kylo-pl-bucket/yaml-cpp-0.5.1_install.tar.bz2
   - tar xjvf yaml-cpp-0.5.1_install.tar.bz2 --strip 1 -C $USERDIR
+  - wget https://github.com/puppetlabs/leatherman/releases/download/${LEATHERMAN_VERSION}/leatherman.tar.gz
```

Comment:
Is there a latest link that works?

How should puppet-agent builds work? I guess they can't really specify a dependency, so we want them to be tagged as well. I can't think of any way to automatically keep them in-sync.